app/views/doorkeeper/applications/_form.html.erb

```diff
@@ -22,7 +22,7 @@
 
       <% if Doorkeeper.configuration.native_redirect_uri %>
           <span class="form-text text-secondary">
-            <%= raw t("doorkeeper.applications.help.native_redirect_uri", native_redirect_uri: content_tag(:code, class: "bg-light") { Doorkeeper.configuration.native_redirect_uri }) %>
+            <%= raw t("doorkeeper.applications.help.native_redirect_uri", native_redirect_uri: tag.code(class: "bg-light") { Doorkeeper.configuration.native_redirect_uri }) %>
```

Comment:
TIL about `tag.#{DOM_TAG}` which is a HTML builder much easier to use than `content_tag`, see https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag

Reply:
Very neat!